Wikipedia/Code/WMFContentGroupDataStore.h

```diff
@@ -1,50 +1,40 @@
-#import "WMFBaseDataStore.h"
-#import "WMFDataSource.h"
-
-@class WMFContentGroup;
-@class WMFRelatedPagesContentGroup;
+#import "WMFContentGroup+Extensions.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface WMFContentGroupDataStore : WMFBaseDataStore
+@interface WMFContentGroupDataStore : NSObject
```

Comment:
Yeah - I think long term that's the solution, but for simplicity on this first pass, keeping them around.